about.html

```diff
@@ -0,0 +1,114 @@
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </div>
+        <div id="main-header">
+            <a href="index.html">           
+                <div id="nig-logo">
+                    <img src="./Images/nigeria.png" class="ng" alt="nigerian map">
+                    <h2 class="summit">N.I.E.S</h2>
+                </div>
+            </a>
+            <nav id="header-nav">
+                <ul id="header-list">
+                    <li class="close">&times;</li>
+                    <li class="list"><a href="./about.html">About</a></li>
+                    <li class="list">Program</li>
+                    <li class="list">Join</li>
+                    <li class="list">Sponsor</li>
+                    <li class="list">News</li>
+                    <li class="list">N.I.E.S '22</li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+    <div id="hamburger">
+        <div class="line"></div>
+        <div class="line"></div>
+        <div class="line"></div>
+    </div>
+    <main id="about-main">
+        <section id="top-section">
+            <p class="first-para">"Promoting Good Electrical Power For A Better Nigeria"</p>
+            <h1 class="topic">The Nigerian International Energy Summit 2022</h1>
+            <div class="hr1"></div>
+            <p class="second-para">In the spirit of the global agenda for energy transition and the need for us to remain reelevant and ahead, The Nigerian International Petroleum Summit quietly did a rebrand. From this moment, it'll be known as The Nigerian International Energy Summit. <br> With the ongoing reforms in the sector and improved engagement with host communities, Nigeria is positioning herself to be the foremost energy investment destination. The official Nigerian International Energy Summit continues to create the perfect platform for the world to meet Nigeria's energy industry players.</p>
+            <p class="third-para">For further information about N.I.E.S 2022, CONTACT US ON: <a href="mailto:iyaraogaga69@gmail.com"><b>iyaraogaga69@gmail.com</b></a></p>
+        </section>
+        <section id="mid-section">
+            <h6 class="heading1">Nigerian International Energy Summit 2022 Logo</h6>
+            <div class="hr2"></div>
+            <p class="fourth-para">The logo for the N.I.E.S 2022 was decided through the logo competition from 8 June to 7 July 2021.</p>
+            <div id="nig-logo3">
+                <img src="./Images/nigeria.png" class="ng" alt="nigerian map">
+                <h2 class="summit">N.I.E.S</h2>
+            </div>
+        </section>
+        <section id="bottom-section">
+            <h2 class="heading2">See The Past N.I.E Summits</h2>
+            <div class="hr1"></div>
+            <p class="fifth-para">Take a look at the last two N.I.E.S which took place in Lagos and Calabar, Nigeria.</p>
```

Comment:
- [ ] In mobile width, the filter of images and the images height do not match, and the filter does not fully cover the images, please address this issue: 
<img width="319" alt="Screenshot 2022-04-13 at 17 38 24" src="https://user-images.githubusercontent.com/77038610/163193110-dabf91bd-4ec4-4205-a505-5247328e529f.png">